schema_utils.py

```diff
@@ -185,6 +204,203 @@ def get_validator(validator_id):
     return _Validators.get(validator_id)
 
 
+def is_numeric_operand(string):
+    """Checks if the given string represents a valid number.
+
+    Args:
+        string: str. The operand to be validated.
+
+    Returns:
+        bool. Whether the given string is a valid numeric operand.
+    """
+    try:
+        float(string)
+        return True
+    except ValueError:
+        return False
+
+
+# Given string must be a singular english alphabet or a valid greek letter.
+IS_ALGEBRAIC_OPERAND = lambda string: (
```

Comment:
This is a function; it seems weird to write it as a constant. is_algebraic_operand perhaps?

Ditto below.

Reply:
Since this is a global variable, the linter treats it as a constant which is why it requires uppercase. Some other instances of this pattern are [here](https://github.com/oppia/oppia/blob/a1fa806b883d4522df07698022d6e5acc9e48b85/feconf.py#L471).

Reply:
Then I would suggest that you write it as a regular function, with args, docstring, etc.

Reply:
Done.